### DISCUSSION TITLE
Port 53 on macos catalina "bind: address already in use"

### DISCUSSION THREAD

**Discussion by dzg** [AdguardTeam/AdGuardHome#2729] [Q&A]
Trying to set this up on macos catalina. Running from terminal with sudo.

No matter which interface I choose, I get "listen udp 192.168.1.55:53: bind: address already in use". Same for 5353.

`sudo lsof -i :53` shows `mdnsresponder` using port 53.

After some digging ... seems port 53 is being used because I have Internet Sharing enabled. Any way to use this with Internet Sharing?

**Comment by ameshkov** [comment]
Tbh, I don't know if that's possible, and this is the first time this was asked here.

Marked as "help wanted", maybe someone can help with this.

**Comment by Kainichy** [comment]
Use this to disable mDNSresponder directly:
`sudo launchctl unload -w /System/Library/LaunchDaemons/com.apple.nDNSresponder.plist`
But it may stop internet sharing. However you can have a try. To enable again:
`sudo launchctl load -w /System/Library/LaunchDaemons/com.apple.nDNSresponder.plist`

**Comment by MrDrache333** [comment]
`Unload failed: 150: Operation not permitted while System Integrity Protection is engaged`
But you also have to disable System Integrity Protection first.
